pyproject.toml

```diff
@@ -35,10 +35,14 @@ dependencies = [
     "MarkupSafe>=1.1",
     "mkdocs>=1.2",
     "mkdocs-autorefs>=0.3.1",
+    "mkdocstrings-python-legacy>=0.2",
     "pymdown-extensions>=6.3",
-    "pytkdocs>=0.14.0",
 ]
 
+[project.optional-dependencies]
+python-legacy = ["mkdocstrings-python-legacy>=0.2"]
+python = ["mkdocstrings-python>=0.5.1"]
```

Comment:
Why do you think these "extras" are necessary or preferred? Why not just tell people to depend on `mkdocstrings-python` rather than `mkdocstrings[python]`? This makes the playing field more fair for new handlers which wouldn't have to be registered here.
Or I don't know...
Maybe it *is* nice, but anyway pointing this out, just in case you didn't realize that this isn't necessary.

Reply:
I know it's not necessary :slightly_smiling_face: 
And I do feel it's nice to have. Example: if you have both Crystal and Python code, you can depend on `mkdocstrings[crystal,python]`.
And I realize we should definitely add the `crystal` extra :smile: 

To me these extras are just a way to advertise some handlers as the *official* ones (could be community contributed handlers, just like yours, handlers from other GitHub accounts, etc.). Any handler can still be used by depending on it directly.

Reply:
OK I'll go ahead and keep the extras, I like the idea, and I don't see any downside (apart from maintenance maybe which I'm willing to take on), let see how it unfolds :slightly_smiling_face: